src/createAPI.js

```diff
@@ -5,8 +5,8 @@ import applyMiddleware from './applyMiddleware';
 const createAPI = (
     resources = {},
     middleware = [],
-    APINamespace = '',
-    fetchOptions = {}
+    APINamespace,
+    fetchOptions
```

Comment:
@TimeRaider 'cause it's overriding default values in `callAPIMethod` and it's not expected behavior